README.rst

```diff
@@ -25,7 +25,7 @@ Version Support
 ---------------
 
 *behave-django* is `tested against`_ the officially supported combinations of
-Python and Django (Django 2.2, 3.1, 3.2 on Python 3.6, 3.7, 3.8, 3.9, 3.10).
+Python and Django (Django 2.2, 3.2, 4.0 on Python 3.6, 3.7, 3.8, 3.9, 3.10).
```

Comment:
I noticed that the badges are still pointing to Travis. Could we change this to show the build status on GitHub Actions?

Reply:
The badges depend on the released version. We would have to publish a new release on PyPI. We could.